src/main/java/org/broadinstitute/hellbender/cmdline/StandardArgumentDefinitions.java

```diff
@@ -11,12 +11,16 @@
     public static final String REFERENCE_LONG_NAME = "reference";
     public static final String VARIANT_LONG_NAME = "variant";
     public static final String FEATURE_LONG_NAME = "feature";
+    public static final String LENIENT_LONG_NAME = "lenient";
+    public static final String INTERVAL_LONG_NAME = "interval";
 
     public static final String INPUT_SHORT_NAME = "I";
     public static final String OUTPUT_SHORT_NAME = "O";
     public static final String REFERENCE_SHORT_NAME = "R";
     public static final String VARIANT_SHORT_NAME = "V";
     public static final String FEATURE_SHORT_NAME = "F";
+    public static final String LENIENT_SHORT_NAME = "LE";
+    public static final String INTERVAL_SHORT_NAME = "IN";
```

Comment:
We should define constants here based on the ones in `IntervalArgumentCollection` though.